docs/support_playbook.md

```diff
@@ -106,3 +114,61 @@ status: 400, body: {"title":"Teacher has no incomplete ITT record","status":400,
 * Cross reference the trainee details on Register with the trainee details on DQT, you can use the DQT API for this - checking the trainee timeline on Register can also be helful
 * If there are no differences, it is likely a race condition and you can delete the failed update job
 * If there are differences, speak to DQT and maybe contact provider to see what needs updating
+
+### Dqt::RetrieveTrnJob for Trainee id: xxx has timed out after 4 days.
+
+We see this error on slack when our polling job times out before we receive a
+TRN from DQT. The default time out is four days from the trainee's
+`submitted_for_trn_at` date.
+
+We need to understand from DQT why the TRN is taking a while to assign. Is there
+a problem we need to address?
```

Comment:
is the step here to go to [twd_register_dqt_integration](https://ukgovernmentdfe.slack.com/archives/C03SV1L444U) and ask? is there anyone in particular who should be @'d ?

Reply:
yup! i don't @- anyone in particular unless it's urgent or not getting a response.. then Rob Collier is your man.